eclipse-theia/theia
# Align `QuickPickItem` declarations

#### What it does


Fixes #10942 by aligning our quick-pick type with VSCode's

#### How to test


1. Download [quick-pick-align-0.0.1.zip](https://github.com/eclipse-theia/theia/files/8365758/quick-pick-align-0.0.1.zip)
2. Run the command 'Show a quick pick'
3. See this:

![image](https://user-images.githubusercontent.com/62660806/160474737-124a236e-7bac-4a61-973e-b3ec18aedb87.png)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
